.github/workflows/pull-requests.yml

```diff
@@ -590,23 +589,232 @@ jobs:
            fi
           done
 
-      - name: Request clusters info
+      - name: Waiting 120s before check nodes health
         run: |
-          sleep 5
-          echo "Dag L1"
-          curl http://127.0.0.1:9100/cluster/info
+          echo "Waiting for 120s before checking cluster individual nodes health..."
+          sleep 120
```

Comment:
I think you can query the health in the loop with 120s timeout instead of fixed delay. That way we won't wait 120s if nodes are ready earlier.